lib/Ix/DBIC/Result.pm

```diff
@@ -329,10 +329,6 @@ sub ix_update_extra_search ($self, $ctx, $arg) {
   );
 }
 
-sub ix_update_extra_select {
```

Comment:
I think you're including two unrelated commits in this mr?

Reply:
Removed.